example/metric/README.md

```diff
@@ -22,7 +22,11 @@ Next, set this project ID to the environment variable `GOOGLE_PROJECT_ID` using
 export GOOGLE_PROJECT_ID=$(gcloud config get-value project)
 ```
 
-Once you ensure the API is enabled, then build the example application and run the executable.
+Once you ensure the API is enabled, then build the example application and run the executable. There are two separate examples showcasing the following - 
+1. Exporting metrics via the SDK.
+2. Exporting metrics via the OpenTelemetry Collector.
+
+Change the current directory to the example you wish to run - either to `sdk` directory or `collector` directory and then run the example using following commands:
```

Comment:
I think the text "SDK" and "OpenTelemetry Collector" in the bullet points should not be a hyperlink to the folders since they are actually intended to explain what component is being used to export metrics. 

I will add hyperlinks in the instructions though.